<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProtoDialog</class>
 <widget class="QDialog" name="ProtoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>327</width>
    <height>293</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_main">
      <attribute name="title">
       <string>通用规约配置</string>
      </attribute>
      <widget class="QComboBox" name="combo_yc">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>181</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>归一化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP56时标的归一化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP24时标的归一化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>标度化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP56时标的标度化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP24时标的标度化值</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>短浮点数</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP56时标的短浮点数</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP24时标的短浮点数</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="TEXT3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>传送原因COT长度：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="combo_yx">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>50</y>
         <width>181</width>
         <height>30</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>单点遥信</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP56时标的单点遥信</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP24时标的单点遥信</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>双点遥信</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP56时标的双点遥信</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>带CP24时标的双点遥信</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="TEXT1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>15</y>
         <width>101</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>遥测值类型：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spin_cot">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>130</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spin_ca">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>90</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>93</y>
         <width>131</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>公共地址CA长度：</string>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>204</y>
         <width>181</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string>数据是否采用压缩格式：</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_zip">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>210</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>启用压缩格式</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSpinBox" name="spin_ioa">
       <property name="geometry">
        <rect>
         <x>179</x>
         <y>170</y>
         <width>111</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>3</number>
       </property>
       <property name="value">
        <number>3</number>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>遥信值类型：</string>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>141</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>信息体地址IOA长度：</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_LinkLayer">
      <attribute name="title">
       <string>链路层配置</string>
      </attribute>
      <widget class="QSpinBox" name="spin_ack_timeout">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>140</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>1000</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="check_char">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>10</y>
         <width>151</width>
         <height>30</height>
        </rect>
       </property>
       <property name="text">
        <string>启用单字节回复</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>107</y>
         <width>271</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>未收到ACK时重复消息传输超时(毫秒)：</string>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT8">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>131</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>使用单字节回复：</string>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT9">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>46</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>规约链路层确认超时(毫秒)：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spin_ack">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>73</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="TEXT11">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>170</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>链路层链路地址长度：</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spin_llength">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>200</y>
         <width>121</width>
         <height>30</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>check_zip</sender>
   <signal>clicked(bool)</signal>
   <receiver>ProtoDialog</receiver>
   <slot>check_zip_click(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>253</x>
     <y>259</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>check_char</sender>
   <signal>clicked(bool)</signal>
   <receiver>ProtoDialog</receiver>
   <slot>check_char_click(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>146</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>check_zip_click(bool)</slot>
  <slot>check_char_click(bool)</slot>
 </slots>
</ui>
